tools/gff3_to_json/gff3_to_json.py

```diff
@@ -62,13 +64,17 @@ def exon_to_json(cols, species):
         'strand': 1 if cols[6] == '+' else -1,
     }
     for attr in cols[8].split(';'):
-        (tag, value) = attr.split('=')
-        if tag == 'ID':
-            exon['id'] = value
-        else:
+        if(len(attr)>0):
+            (tag, value) = attr.split('=')
+
+            tag = tag.lower()
```

Comment:
GFF3-compliant attributes with predefined meaning are case-sensitive (see https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md ), I'm going to revert this change.